frontend/app/components/links.tsx

```diff
@@ -176,7 +176,7 @@ function scrollAndFocusFromAnchorLink(href: string): void {
     const { hash } = new URL(href);
 
     if (hash) {
-      const targetElement = document.querySelector<HTMLElement>(hash);
+      const targetElement = document.getElementById(hash.replace('#', ''));
```

Comment:
Can't use `querySelector` with `hash` when `id` is generated by `useId()`. You would have to use `CSS.escape()` to make it work. I decided to use `getElementById` instead.

Related [issue](https://github.com/facebook/react/issues/28404)